student.rb

```diff
@@ -1,6 +1,9 @@
 require './person'
+require './classroom'
 
 class Student < Person
+  attr_accessor :classroom
+
   def initialize(*person, classroom)
```

Comment:
- [ ] Please make sure when  setting the classroom for a student it also adds it to the classrooms' students:

you can try that code snippet in initialize  method:
``classroom.students.push(self) unless classroom.students.include?(self)``

Reply:
ok i add this snippet to the student class